### ISSUE TITLE
Errors during Saros/E release build

### ISSUE THREAD

**Issue by m273d15** [saros-project/saros#463]
If you create a Saros/E release (create update site or dropin) by using the eclipse PDE plugin, several errors occurs, because JUnit is not provided during release build. See log-file below.
JUnit and powermock are required during the release build, because the STF framework is build with the Saros/E component. If #447 is merged this issue does not exist anymore, because the PR separates Saros/E and STF.

[saros_eclipse.log](https://github.com/saros-project/saros/files/2929766/saros_eclipse.log)

**Comment by m273d15** [comment]
Implementing a workaround would increase the complexity of the build process and make the whole build description and process ugly. Therefore we should simply use a manual workaround:
1. create a new directory (e.g. `releaseLibs`) in the `eclipse/` directory
2. execute `./gradlew :saros.core:testJar :saros.ui:testJar` (to create the jars which are required for **3.**)
3. copy the jars `core/build/libs/saros.core-tests.jar` and `ui/build/libs/saros.ui-tests.jar` into the directory created in **1.** 
4. copy all tests dependency jars (junit, powermock, easymock; you will find them in your gradle cache)   into the directory created in **1.**
5. Add all files which are contained in the directory as **Extra Classpath Entries** of the `eclipse/build.properties` file. In my case I had to add the following snipped to my configuration:

```properties
jars.extra.classpath = releaseLibs/junit-4.11.jar,\
                       releaseLibs/powermock-core-1.5.1.jar,\
                       releaseLibs/powermock-api-easymock-1.5.1.jar,\
                       releaseLibs/powermock-api-support-1.5.1.jar,\
                       releaseLibs/powermock-module-junit4-1.5.1.jar,\
                       releaseLibs/powermock-module-junit4-common-1.5.1.jar,\
                       releaseLibs/powermock-module-junit4-legacy-1.5.1.jar,\
                       releaseLibs/powermock-reflect-1.5.1.jar,\
                       releaseLibs/saros.core-tests.jar,\
                       releaseLibs/saros.ui-tests.jar,\
                       releaseLibs/easymock-3.1.jar
```

**Comment by m273d15** [comment]
The merge of solved #489 (+ #503) this issue.
